geode-protobuf/src/main/proto/org/apache/geode/internal/protocol/protobuf/v1/clientProtocol.proto

```diff
@@ -19,13 +19,13 @@
 */
 
 syntax = "proto3";
-package org.apache.geode.internal.protocol.protobuf;
+package org.apache.geode.internal.protocol.protobuf.v1;
 
 import "google/protobuf/any.proto";
-import "region_API.proto";
-import "server_API.proto";
-import "basicTypes.proto";
-import "connection_API.proto";
+import "org/apache/geode/internal/protocol/protobuf/v1/region_API.proto";
```

Comment:
I don't understand why the proto file location has to be fully qualified. I've tested and just having the files reside in `v1/*proto` works